src/masonite/foundation/Kernel.py

```diff
@@ -91,6 +93,7 @@ def register_framework(self):
             "sign", Sign("-RkDOqXojJIlsF_I8wWiUq_KRZ0PtGWTOZ676u5HtLg=")
         )
         self.application.bind("base_url", "http://localhost:8000")
+        self.application.bind("resolver", DB)
```

Comment:
@josephmancuso Why is it called `resolver` ? Is it bound only for being used in tests, or do you plan that we can resolve this in other parts of the app ?